CHANGELOG.md

```diff
@@ -46,6 +46,12 @@
   * "setCustom(String, Object)" instead use "Countly::userProfile::setProperty" via "instance()" call
   * "set(String, Object)" instead use "Countly::userProfile::setProperty" via "instance()" call
   * "picture(byte[])" instead use "Countly::userProfile::setProperty" via "instance()" call
+* Deprecated "View::start(bool)" call, use "Countly::views::startView" instead via "instance()" call.

```

Comment:
it was not an autostopped view? at least if the same name was used?

Reply:
It was an autostopped one, View interface exposes it to used by Countly and SessionImpl classes. But if a user creates a View object, they can trigger it. But sdk starts them automatically when a Countly.view or Countly.session.view is called

Reply:
For other yeap, there is no same name for them